### PR TITLE
Fix link fragment parsing issue in PDF build

### PR DESCRIPTION
**Pull request by rejahrehim**
- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix parsing issues in link fragments (PDF build)

### CODE REVIEW

File: pdf/scripts/make-pdf.sh [hunk 1]
```diff
@@ -37,11 +37,10 @@ echo "<img src=\"images/back-cover.png\" style=\"overflow:hidden; margin-bottom:
 
 # Create the single document Markdown file
 # Sed section 1: Add page break after each chapter
-# Sed section 2: Correct Markdown file links with fragment identifiers. Remove file path and keep fragment identifier alone.
 ls build/md | sort -n | while read x; do cat build/md/$x | sed -e 's/^# /<div style=\"page-break-after: always\;\"><\/div>\
 \
-# /' | sed 's/\[\([^\n]\+\)\]([^\n]\+.md#\([^\)]\+\)/[\1](#\2/' | \
-# Sed section 3 - 8: Replace internal Links inside headings with anchor tags inside the respective heading and link href as the heading text
+# /' |
```
Comment (kingthorin) [inline pdf/scripts/make-pdf.sh:42]: Remove the whole commented line?

File: pdf/scripts/make-pdf.sh [hunk 2]
```diff
@@ -103,7 +106,7 @@ pdftk build/cover-$VERSION.pdf build/second-cover-$VERSION.pdf build/wstg-doc-$V
 # Sed and Python sections are exactly same as the previous one
 ls build/md | sort -n | while read x; do cat build/md/$x | sed -e 's/^# /<div style=\"page-break-after: always\;\"><\/div>\
 \
-# /' | sed 's/\[\([^\n]\+\)\]([^\n]\+.md#\([^\)]\+\)/[\1](#\2/' | \
+# /' |
```
Comment (kingthorin) [inline pdf/scripts/make-pdf.sh:109]: Remove the whole commented line?

File: pdf/scripts/make-pdf.sh (hunk 1 above)
Comment (rejahrehim) [reply pdf/scripts/make-pdf.sh:42]: Its not commented line, Its part of the sed command.

File: pdf/scripts/make-pdf.sh (hunk 2 above)
Comment (rejahrehim) [reply pdf/scripts/make-pdf.sh:109]: Its not commented line, Its part of the sed command.

File: pdf/scripts/make-pdf.sh (hunk 1 above)
Comment (kingthorin) [reply pdf/scripts/make-pdf.sh:42]: Okay

File: pdf/scripts/make-pdf.sh (hunk 2 above)
Comment (kingthorin) [reply pdf/scripts/make-pdf.sh:109]: Okay

**Pull request merged**